### DISCUSSION TITLE
Info regarding the logicle transformation

### DISCUSSION THREAD

**Discussion by 5ittn3r** [whitews/FlowKit#204] [General]
Hi,
I applied the logicle transformation to my data.
My fluorescence channels'intensities are now all in the range of 0.0-1.0.
logicle data shouldn't be bound to this range. Is it normalized? if so, how and why?
All the best,
J

**Comment by whitews** [comment]
Hi @5ittn3r

The [GatingML 2.0 specification](https://flowcyt.sourceforge.net/gating/latest.pdf) describes the Logicle transform better than I can here. See Section 6.5.1 where the transform and its parameters are defined. Section 6.5.4 is also helpful as it contains a table showing the input and output values for several sets of parameters. Parameter T controls the top of the scale that is mapped to 1. Input values beyond T are allowed and will map to values greater than 1.

Regards,
Scott
